cmd/vulcan-trivy/README.md

```diff
@@ -0,0 +1,2 @@
+# Vulcan Trivy Check
+Utility used: [trivy](https://aquasecurity.github.io/trivy/v0.32/)
```

Comment:
No strong opinion, but I would go with something like this

```
# vulcan-trivy

Checks if a Docker image uses vulnerable packages or dependencies using Trivy

Based on [trivy](https://aquasecurity.github.io/trivy)
```

Reply:
I use this style now. Thanks!